client/src/main/java/com/paypal/selion/reports/reporter/excelreport/ExcelReport.java

```diff
@@ -232,8 +255,16 @@ private void createReportMap() {
         BaseReport<List<String>> bR = (BaseReport<List<String>>) fullReportMap.get(fullReportMap.size() - 1)
                 .getGeneratedReport().iterator().next();
         List<String> lsTitles = Arrays.asList(new String[] { "Class Name", "Method/Testcase id", "Test Description",
-                "Group[s]", "Time taken", "Link", "Error Message", "Error Details" });
+                "Group[s]", "Time taken", "Output Logs", "Error Message", "Error Details" });
         bR.setColTitles(lsTitles);
+        
+        // TestCase Output Details Report
+        Map<String, List<List<String>>> fifthTestOutputSubReportMap = new LinkedHashMap<String, List<List<String>>>();
+        fifthTestOutputSubReportMap.put("Tests Output", tcOutputData);
```

Comment:
Addressed: This is the section header. Changed this to 'Test Output'. Will also update ReportSheetNames#TESTOUTPUTDETAILSREPORT to use 'Test Output'.